### DISCUSSION TITLE
When we draw, why do we draw with colors instead of symbols?

### DISCUSSION THREAD

**Discussion by AhmetPolat43** [OpenOrienteering/mapper#1797] [Questions & Answers] [answered]
Why do we use a pencil to draw the whole map with orienteering symbols? Why don't we draw them all with symbols as they are?
![93875395-307db600-fcd5-11ea-8e04-d673d51c74c2](https://user-images.githubusercontent.com/62382116/102110666-7f576980-3e46-11eb-9a23-361a41236633.png)

**Comment by mlerjen** [comment, marked as answer]
Because longer line and larger area features which cannot be drawn right away are much faster drawn once finished (e.g. contours, area features).  Everything with appending and snapping, everything you need Ctrl, Enter or Shift for, is faster and cleaner mapped at the desktop. This is what you would want to do: delivering a clean drawing.

**Comment by AhmetPolat43** [comment]
Wouldn't it be more logical to draw symbols with the free drawing option instead?

**Comment by AhmetPolat43** [comment]
![Uploading Screenshot_20201214-211109_OpenOrienteering Mapper.jpg…]()

**Comment by dg0yt** [comment]
> Wouldn't it be more logical to draw symbols with the free drawing option instead?

You can do that if you want. It is also a matter of personal taste.

**Comment by dg0yt** [comment]
(Moving this "issue" to our new "discussions".)
